README.md

```diff
@@ -1,63 +1,200 @@
+
+First, you're going to need a way to store the names of the songs that make up the _Dirty Dancing_ soundtrack. It makes sense to store them in a property, doesn't it? And since it's a list of songs, an `Array` of `String`s (`[String]`) seems like an appropriate data type. Create a property called `dirtyDancingSoundtrack` to store these items. The songs that comprise the _Dirty Dancing_ soundtrack are:
+
+1. "(I've Had) The Time of My Life"
+2. "Be My Baby"
+3. "She's Like the Wind"
+4. "Hungry Eyes"
+5. "STAY"
+6. "Yes"
+7. "You Don't Own Me"
+8. "Hey Baby"
+9. "Overload"
+10. "Love Is Strange"
+11. "Where Are You Tonight?"
+12. "In the Still of the Night"
+
+Did you create that property? If you need some help, here's what `TableViewController`'s class definition should look like so far:
+
+```swift
+class TableViewController: UITableViewController {
+    let dirtyDancingSoundtrack = [
+        "(I've Had) The Time of My Life",
+        "Be My Baby",
+        "She's Like the Wind",
+        "Hungry Eyes",
+        "STAY",
+        "Yes",
+        "You Don't Own Me",
+        "Hey Baby",
+        "Overload",
+        "Love Is Strange",
+        "Where Are You Tonight?",
+        "In the Still of the Night"
+    ]
+}
+```
+
+(Note that there are a few ways to create properties in Swift. If your `dirtyDancingSoundtrack` property is defined a bit differently, that's okay!)
+
+Now you can fill in the methods that will provide data back to your table view. Do you remember the three methods you should provide so the table view can be drawn correctly?
+
+The first is `numberOfSectionsInTableView(_:)`, which returns the number of _sections_ in the table view. In this relatively simple app, there is only one section, so this method can return 1:
+
+```swift
+override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return 1
+}
+```
+
+The second method you must provide is `tableView(_:numberOfRowsInSection:)`, which returns the number of _rows_ for a given section. There's only one section, so you don't have to worry about the section in this method. There is one row for each song, so you can just return the total number of songs:
+
+```swift
+override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return dirtyDancingSoundtrack.count
+}
+```
+
+Finally, the trickiest of the three methods: `tableView(_:cellForRowAtIndexPath:)`. Remember, this method passes in an exact section (which is always `0` in this app, since there's only one section) and row number, and is responsible for returning a rendered _cell_ corresponding to that row. This method is a bit tricky because you have to worry about _dequeuing_ a cell (do you remember why from the previous lesson) and setting its text label correctly.
```

Comment:
The prior lesson didn't talk about this. I kind of added a thing in the prior reading (which you can check out) that touches on the row / section you can access from the `indexPath` but I don't specifically talk about the deque / reuseidzntifier part. We might have to go back and add that.